types/redux-batched-subscribe/redux-batched-subscribe-tests.ts

```diff
@@ -0,0 +1,15 @@
+import { createStore, Store, Reducer } from "redux";
+import { batchedSubscribe } from 'redux-batched-subscribe';
+
+interface State {
+    [key: string]: any;
+}
+
+const rootReducer: Reducer<State> = () => ({});
+declare const batchFunction: () => void;
```

Comment:
Could you use an example of a batch function? Of the form `notify => { ... something using notify ... }`